backend/src/controllers/tasks.controller.ts

```diff
@@ -13,9 +21,25 @@ export class TasksController {
 
   @Post()
   insertTask(@Body() task: Task): Promise<void> {
+    if (task.effort < 0) {
+      console.log('Client sent a negative effort value for a new task.');
+      return;
+    }
     return this.tasksService.insertTask(task);
   }
 
+  @Put('/:taskId')
+  updateTask(
+    @Param('taskId') taskId: number,
+    @Body() task: Task,
+  ): Promise<void> {
+    if (task.effort < 0) {
+      console.log('Client sent a negative effort value for task id: ' + taskId);
+      return;
```

Comment:
As mentioned above, HTTP status code `400 Bad Request` here can better communicate the error to the clients